doc/user_documentation.rst

```diff
@@ -32,6 +32,22 @@ How to enable copr repository?
 
 :ref:`how_to_enable_repo`
 
+
+Public Copr instances
+---------------------
+
+Copr is a free software and anyone can maintain their own instance in
+case the Fedora Copr instance doesn't suit their needs. This is a list
+of known Copr instances:
+
+- https://copr.fedorainfracloud.org - Fedora Copr instance which is
+  also considered to be the "main" Copr instance.
```

Comment:
I'd say *which is the oldest instance, and also considered to be the default Copr instance by many users and tools*

Reply:
I think it would be nice to cite a public issue tracker, or something alike ...

Reply:
Good idea. So what about a table like this?

![Screenshot_2023-06-01_00-18-42](https://github.com/fedora-copr/copr/assets/2151771/55e820b5-14d4-4a2f-be29-064787c506ae)